manager/controllers/default/system/dashboards/update.class.php

```diff
@@ -58,6 +58,11 @@ public function getWidgets() {
         /** @var modDashboardWidgetPlacement $placement */
         foreach ($placements as $placement) {
             $placement->getOne('Widget');
+
+            if ($placement->Widget == null) {
```

Comment:
`if (is_null($placement))` looks shorter and more clear and safe, or need to use `===`

Reply:
I believe this should make sure it is an instance of modDashboardWidget instead of checking if not null.